tests/core/test_variable.py

```diff
@@ -6,15 +6,15 @@
 import ixmp4
 from ixmp4 import Variable
```

Comment:
(Just to keep an overview: same note about imports would apply here.)
(Curious: here, this import structure is already on main like this, it seems.)